src/pages/files.js

```diff
@@ -72,22 +72,34 @@ export default class Files extends React.Component {
     }
   };
 
+  saveFileToS3() {
+    const { files } = this.state;
+
+    const ids = Object.keys(files);
+
+    return postJson('/files/save', { ids }).then(response => response.json());
```

Comment:
This is breaking. `response.json()` has already been called in `fetchJson` method.

Reply:
Yes, noticed that, my bad. Feel free to add the fix.